code/datums/emotes.dm

```diff
@@ -62,7 +63,15 @@
 	var/dchatmsg = "<b>[user]</b> [msg]"
 
 	var/tmp_sound = get_sound(user)
-	if(tmp_sound && (!only_forced_audio || !intentional))
+	var/play_emote_audio = TRUE
+	// Only check if we can play a sound if there's actually a sound to play.
+	if(tmp_sound != null)
+		if(!user.audio_emotes_used)
+			user.audio_emotes_used = list()
+		else if(user.audio_emotes_used[src] + audio_cooldown > world.time)
+			play_emote_audio = FALSE
+		user.audio_emotes_used[src] = world.time
+	if(tmp_sound && play_emote_audio && (!only_forced_audio || !intentional))
```

Comment:
Sorry, I told you to use lazy list macros, but I didn't take a proper look at this.
These are cooldowns, not just lazy lists. We have other macros for these:

```suggestion
	if(tmp_sound && (!only_forced_audio || !intentional) && !TIMER_COOLDOWN_CHECK(user, type))
		TIMER_COOLDOWN_START(user, type, audio_cooldown)
```